packages/internal/src/marshal.js

```diff
@@ -72,7 +72,7 @@ harden(assertCapData);
  * @param {string} key
  * @param {ReturnType<typeof import('@endo/marshal').makeMarshal>['fromCapData']} fromCapData
  * @param {number} index index of the desired value in a deserialized stream cell
- * @return {any}
+ * @returns {any}
```

Comment:
This is a drive by, totally unrelated to this PR. I just happened to notice a lint warning.